lib/GADS/PeopleFilter.pm

```diff
@@ -0,0 +1,43 @@
+use utf8;
+
+package GADS::PeopleFilter;
+
+use strict;
+use warnings;
+
+use Moo;
+use Log::Report 'linkspace';
+use Encode;
+use MIME::Base64;
+
+extends 'GADS::Filter';
+
+sub base64 {
```

Comment:
I'm not sure that this whole function is needed and/or the one in the parent class could be used/combined?